src/profile/forms/Certificates.jsx

```diff
@@ -98,7 +98,12 @@ class Certificates extends React.Component {
                 id="profile.certificate.completion.date.label"
                 defaultMessage="Completed on {date}"
                 values={{
-                  date: <FormattedDate value={new Date(modifiedDate)} />,
+                  date: (
+                    <FormattedDate
+                      value={new Date(modifiedDate)}
+                      timeZone="utc"
```

Comment:
Due to timezone difference when being executed from some other region tests may not pass locally as was the case with me. Added the `timeZone` props so the time will always be evaluated in utc timezone and hence tests will run consistently on CI as well as locally.